config/testgrids/kubernetes/presubmits/config.yaml

```
@@ -147,6 +147,8 @@ dashboards:
     base_options: width=10
   - name: pull-kubernetes-node-e2e
     test_group_name: pull-kubernetes-node-e2e
+  - name: pull-kubernetes-node-e2e-kubetest2
+    test_group_name: pull-kubernetes-node-e2e-kubetest2
```

May as well go ahead and add this if you're adding it here to be able to do so
```suggestion
    test_group_name: pull-kubernetes-node-e2e-kubetest2
    base_options: width=10
```